src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -448,8 +447,20 @@ private void updateLevelButton()
      */
     private void confirmClicked(Button button)
     {
-        MineColonies.getNetwork().sendToServer(new BuildToolPlaceMessage(hutDec.get(hutDecIndex),
-                                                                          getStyles().get(styleIndex), this.pos, rotation, Settings.instance.isInHutMode()));
+        final InventoryPlayer inventory = this.mc.thePlayer.inventory;
```

Comment:
To elaborate, I think that these checks should be done in the handleHut method inside of BuildToolPlaceMessage. Interestingly it looks like there already is some checks there, so those should have prevented the crash. (Maybe they should be improved)

Also sending the message to the player can happen in the same handleHut method.